README.md

```diff
@@ -7,3 +7,15 @@ This is a suitcase subpackage for writing a particular file format.
 ```
 pip install suitcase-msgpack
 ```
+
+## Quick Start
+
+```
+import suitcase.msgpack
+docs = db[-1].documents(fill=True)
+suitcase.msgpack.export(docs, 'my_exported_files/', file_prefix='PREFIX-')
```

Comment:
Why use `PREFIX-`? This example would work for one run, but if you then export a second run it would try to create a file with an identical name, and suitcase would raise an error, refusing to overwrite the existing file.

Reply:
What about `file_prefix='{start[uid]}'` in example?

Reply:
That seems sensible. Is that the default value? I think it is a judgement call whether it is better spell out the default value explicitly like that or to just simplify the example by omitting `file_prefix` entirely. The first option gives more information up front; the second option may be more inviting. Any opinions on this, @jklynch ?

Reply:
Right, that's default.
I vote for omitting it.

Reply:
What about removing `file_prefix=...` from the example and adding some after the example similar to (but correct):

```
The exported file will have path `my_exported_files/1234abcd-blah-blah.msg`. The default file prefix can be changed with the `file_prefix` keyword argument. See <link to documentation>.
```

Reply:
I like that.

Reply:
work for me